src/test/java/samples/ios/IOSBasicInteractionsTest.java

```diff
@@ -0,0 +1,53 @@
+package samples.ios;
+
+import aquality.appium.application.ApplicationManager;
+import io.appium.java_client.ios.IOSDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import samples.ios.testapp.screens.MainScreen;
+import samples.ios.testapp.screens.TwoItemsAlertScreen;
+
+public class IOSBasicInteractionsTest {
+    private IOSDriver<?> driver;
+
+    @BeforeClass
+    public void setUp() {
+        driver = (IOSDriver<?>) ApplicationManager.getApplication().getDriver();
+    }
+
+    @AfterClass
+    public void tearDown() {
+        ApplicationManager.getApplication().quit();
+    }
+
+    @Test
+    public void testSendKeysToInput() {
```

Comment:
please, rename all the tests to patter like:
testShouldBePossibleToSendTextToElement()

do we really need MainScreen for such tests? Can we use elements factory directly here?